apps/api/src/server/applications/s51advice/s51-advice.routes.js

```diff
@@ -26,4 +26,24 @@ router.post(
 	asyncHandler(createS51Advice)
 );
 
+router.get(
+	'/:id',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/applications/s51-advice/{id}'
+        #swagger.description = 'S51 advice ID'
+        #swagger.parameters['id'] = {
+            in: 'path',
+			description: 'S51 advice ID',
+			required: true,
+			type: 'integer'
+        }
+        #swagger.responses[200] = {
+            description: 'List of examination timetable items',
+            schema: { $ref: '#/definitions/S51AdviceResponseBody' }
```

Comment:
I cannot see a matching definition for S51AdviceResponseBody declared in the swagger.js script - though I did not that there was also missing ones from the existing endpoint for POST to create a single advice.  Also called S51AdviceResponseBody.  I added new definitions S51AdviceCreateRequestBody and S51AdviceCreateResponseBody in my ticket for the create POST.  happy to chat about this.